lib/rules/jsx-no-bind.js

```diff
@@ -122,8 +123,8 @@ module.exports = {
       blockVariableNameSets[blockStart][violationType].add(variableName);
     }
 
-    function getBlockStatementAncestors(node) {
-      return context.getAncestors(node).filter(
+    function getBlockStatementAncestors() {
+      return context.getAncestors().filter(
```

Comment:
i'm pretty sure this change will cause eslint 9 to break. can you confirm?

Reply:
Sorry. I do not understand it well.
Should I add v9 to eslint in package.json as shown below and see if I get a compile error?
```
  "devDependencies": {
    "eslint": "^3 || ^4 || ^5 || ^6 || ^7 || ^8 || ^9",
    ...
  },
  "peerDependencies": {
    "eslint": "^3 || ^4 || ^5 || ^6 || ^7 || ^8 || ^9"
  },
```

Reply:
No, adding eslint 9 is a hard task. What i mean is, what’s wrong with passing the node here?

Reply:
When looking at the type information for context.getAncestors, it seems that the argument is unnecessary.
``` ts
(method) Rule.RuleContext.getAncestors(): Node[]
```